topiary-core/src/atom_collection.rs

```diff
@@ -612,34 +646,32 @@ impl AtomCollection {
     ///
     /// A new atom after expanding the softline if applicable.
     fn expand_multiline(&self, atom: Atom, node: &Node) -> Atom {
-        if let Atom::Softline { spaced } = atom {
-            if let Some(parent) = node.parent() {
-                let parent_id = parent.id();
-
-                if self.multi_line_nodes.contains(&parent_id) {
-                    log::debug!(
-                        "Expanding softline to hardline in node {} with parent {}: {}",
-                        node.display_one_based(),
-                        parent_id,
-                        parent.display_one_based()
-                    );
-                    Atom::Hardline
-                } else if spaced {
-                    log::debug!(
-                        "Expanding softline to space in node {} with parent {}: {}",
-                        node.display_one_based(),
-                        parent_id,
-                        parent.display_one_based()
-                    );
-                    Atom::Space
-                } else {
-                    Atom::Empty
-                }
-            } else {
-                Atom::Empty
-            }
+        let Atom::Softline { spaced } = atom else {
+            return atom;
+        };
+        let Some(parent) = node.parent() else {
+            return Atom::Empty;
+        };
+        let parent_id = parent.id();
+
+        if self.multi_line_nodes.contains(&parent_id) {
+            log::debug!(
+                parent_id;
```

Comment:
this does tracing style key/value logging hence the activation of the `kv` feature flag in `log`, happy to remove. Made debug logs a bit cleaner for me.